tools/release/release-verify.py

```diff
@@ -325,6 +354,9 @@ def main():
     # Check that only qualified branches exist in each library repo.
     validate_branches(repo_paths)
 
+    # Check that library repos have a tag and release for each specified version.
+    validate_tags_and_releases(repo_paths, configs)
```

Comment:
I think it will be better to add cmdl option to turn this off if required. It will be helpful if we ever go back to releasing libraries with CSDK.